<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouterKeygen</class>
 <widget class="QMainWindow" name="RouterKeygen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RouterKeygen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelSSID">
      <property name="text">
       <string>SSID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="inputSSID"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="calcButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
